bdsx/launcher.ts

```diff
@@ -93,6 +95,25 @@ function patchForStdio():void {
     }, void_t, {onError:asmcode.jsend_returnZero, name:'bedrockLogNp'}, int32_t, StaticPointer, int64_as_float_t);
     procHacker.write('BedrockLogOut', 0, asm().jmp64(asmcode.logHook, Register.rax));
 
+    procHacker.hookingRawWithCallOriginal('Command::run',
+        makefunc.np((commands, origin, output)=>{
+            if (commandOutputData.has(origin.uuid)) {
+                output.constructAs(CommandOutputType.DataSet);
```

Comment:
it can make memory leaks.
the constructor will overwrite allocated fields without destruction.

Reply:
Should I implement the destructor, or only edit the `type` field of `CommandOutput`?